src/core/cgroup.h

```diff
@@ -159,6 +159,9 @@ struct CGroupContext {
         char **ip_filters_egress;
         LIST_HEAD(CGroupBPFForeignProgram, bpf_foreign_programs);
 
+        Set *restrict_network_interfaces;
+        bool restrict_network_interfaces_is_allow_list:1;
```

Comment:
the ":1" is a bit pointless, given then is nothing to pack this with, maybe just drop it?